core/src/com/unciv/models/ruleset/Difficulty.kt

```diff
@@ -31,36 +34,46 @@ class Difficulty: INamed {
     var aiUnhappinessModifier = 1f
     var turnBarbariansCanEnterPlayerTiles = 0
     var clearBarbarianCampReward = 25
-    
-    fun getDescription(): String {
-        val lines = ArrayList<String>()
-        lines += "Player settings"
-        lines += " - {Base Happiness}: $baseHappiness"
-        lines += " - {Happiness per luxury}: $extraHappinessPerLuxury"
-        lines += " - {Research cost modifier}: $researchCostModifier"
-        lines += " - {Unit cost modifier}: $researchCostModifier"
-        lines += " - {Building cost modifier}: $buildingCostModifier"
-        lines += " - {Policy cost modifier}: $policyCostModifier"
-        lines += " - {Unhappiness modifier}: $unhappinessModifier"
-        lines += " - {Bonus vs. Barbarians}: $barbarianBonus"
-//        lines += " - {Starting units}: $startingUnits"
-        lines += ""
-        lines += "AI settings"
-        lines += " - {AI city growth modifier}: $aiCityGrowthModifier"
-        lines += " - {AI unit cost modifier}: $aiUnitCostModifier"
-        lines += " - {AI building cost modifier}: $aiBuildingCostModifier"
-        lines += " - {AI wonder cost modifier}: $aiWonderCostModifier"
-        lines += " - {AI building maintenance modifier}: $aiBuildingMaintenanceModifier"
-        lines += " - {AI unit maintenance modifier}: $aiUnitMaintenanceModifier"
-//        lines += " - {AI free techs}: $aiFreeTechs"
-//        lines += " - {AI major civilizations starting units}: $aiMajorCivStartingUnits"
-//        lines += " - {AI city-state starting units}: $aiCityStateStartingUnits"
-        lines += " - {AI unhappiness modifier}: $aiUnhappinessModifier"
-        lines += ""
-        lines += "{Turns until barbarians enter player tiles}: $turnBarbariansCanEnterPlayerTiles"
-        lines += "{Gold reward for clearing barbarian camps}: $clearBarbarianCampReward"
 
-        return lines.joinToString("\n") { it.tr() }
+    override var civilopediaText = listOf<FormattedLine>()
+
+//    fun getDescription(): String {
+//        return getCivilopediaTextLines(RulesetCache.getBaseRuleset()).joinToString("\n") { it.text.tr() }
+//    }
```

Comment:
Clean dead lines :)

Reply:
Ah, I abused this as test bed. Sorry. Out they go.